packages/@react-spectrum/textfield/src/TextArea.tsx

```diff
@@ -28,6 +29,9 @@ function TextArea(props: SpectrumTextFieldProps, ref: RefObject<TextFieldRef>) {
     ...otherProps
   } = props;
 
+  // not in stately because this is so we know when to re-measure, which is a spectrum design
+  let [inputValue, setInputValue] = useControlledState(props.value, props.defaultValue, () => {});
```

Comment:
Should we make the textarea controlled based on this?

Reply:
I don't think it matters? I was mostly using it out of convenience to collapse value/defaultValue/none